process/superconducting_tf_coil.py

```diff
@@ -1178,12 +1180,22 @@ def supercon(
             tmarg = t_zero_margin - thelium
             tfcoil_variables.temp_margin = tmarg
 
-        #  Find the current density limited by the protection limit
-        #  (N.B. Unclear of this routine's relevance for Bi-2212 (isumat=2), due
-        #  to presence of fcu argument, which is not used for this model above)
-
+        # Find the current density limited by the protection limit
+        # At present only valid for LTS windings (Nb3Sn properties assumed)
         tfcoil_variables.jwdgpro, vd = self.protect(
```

Comment:
I have preserved the same method name and signature are previously, but given the choice I would split this into two separate calculations, and probably rename the method to something more sensible.